Source/Resources/Content/PowerShell/ExchangeOnline/OrganizationConfig/Configuration.json

```diff
@@ -83,4 +86,4 @@
   "WebPushNotificationsDisabled": false,
   "WebSuggestedRepliesDisabled": false,
   "WorkspaceTenantEnabled": true
-}
\ No newline at end of file
+}

```

Comment:
You are adding a line ending char to the last line.
If you format in VSCode it should remove it again

Reply:
Just applied your suggestion. did it work?